### ISSUE TITLE
BUG? Exception when create EdmModel V7.1.0

### ISSUE THREAD

**Issue by xiaoxin1983** [OData/odata.net#789]
Create EdmModel throw exception

`EdmModel model = new EdmModel();`

System.TypeInitializationException:“The type initializer for 'Microsoft.OData.Edm.Vocabularies.V1.CoreVocabularyModel' threw an exception.”

Internal exception:
ArgumentNullException: Value cannot be null.

### Assemblies affected
ODatalib 7.1.0
.NetCore 1.1

**Comment by matjazmav** [comment]
Getting same exception on same environment.

**Comment by joshcomley** [comment]
I have exactly the same problem. Brand new project, one line of code:

    EdmModel model = new EdmModel();

I get the exception.

**Comment by joshcomley** [comment]
This only happens when using the the package in a .NET Core context. It works fine in standard .NET.

To test this, simply create a new .NET Core Console app, add the reference and try to create a new `EdmModel` instance.

**Comment by AlanWong-MS** [comment]
Thanks for the report. I'll take a look at this.

**Comment by robward-ms** [comment]
Yep, totally a bug on our side. I've pulled 7.1.0 from Nuget.org and replace it with 7.1.1 which contains the Portable libs for Profile 259 but no longer contains the .Net Core libs. We'll post a patch to GitHub soon and make the .Net Core libs available via MyGet.

**Comment by joshcomley** [comment]
Thanks for confirming this, look forward to the fix :)

**Comment by robward-ms** [comment]
You can find a nightly build Nuget package with an fix for this issue here (anything marked 7.2.0): https://www.myget.org/feed/Packages/odlnightly

**Comment by AlanWong-MS** [comment]
Closing this issue for now. We're actively working on a set of validations to ensure the official 7.2.0 doesn't come across this sort of problem. As robward-ms mentioned, you can check out our nightly build that contains the .NET Standard support if you'd like to try the libraries out early.

Thanks!

**Comment by TehWardy** [comment]
Given this was closed a year ago and we still don't have an official 7.2.o from what i can tell on github is this still an issue?
I'm getting this on a new project I created where I imported an existing working OData setup.
Same code just moved it ...

**Comment by xuzhg** [comment]
@TehWardy The latest version of OData lib is [7.5.3](https://www.nuget.org/packages/Microsoft.OData.Core/7.5.3) today. 

I tested the latest version by creating a .Net core console application. Here's my csproj:
```C#
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.1</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.OData.Core" Version="7.5.3" />
  </ItemGroup>

</Project>
```
With the following codes:
```C#
class Program
    {
        static void Main(string[] args)
        {
            EdmModel model = new EdmModel();

            Console.WriteLine("Ok");
        }
    }
```
I can't get the error message and run the application fine. 
Would you please share me your concern?

**Comment by TehWardy** [comment]
So I did some digging.
It looks like from scratch there's an assembly / package for "System.Xml.readerWriter" that's needed by the vocabularies or something and adding the main WebApi Odata package doesn't include that so that's why I was getting the exception.

I have been able to confirm that I was able to get multiple graphs in the same app working too which is cool.
As a side note ... is there a way to link graphs ... sort of like having a "weak foreign key" between them?
